modules/ROOT/pages/manage-host-certificates.adoc

```diff
@@ -0,0 +1,92 @@
+= Managing Host Certificates
+:page-aliases: configure-global-as2-settings.adoc
+
+Anypoint Partner Manager supports the following types of host certificates:
+
+[%header%autowidth.spread]
+|===
+|Certificate Type |Description | Used by
+| Identity key a| Private key used for authentication a| The following types of endpoints, when the authentication type is `Identity Key` or `Basic and Identity Key`:
+
+* SFTP Receive from Partners endpoints owned by the host
+* SFTP Source at Host endpoints
+* SFTP Send to Partners endpoints owned by the host
+* SFTP Target at Host endpoints
+| Public certificate | Certificate generated by a certificate authority (CA) a| 
+* HTTPS Send to Partners endpoints owned by the host, when the TLS context is set to `Use a self-signed certificate`
+* HTTPS Target at Host endpoints, when the TLS context is set to `Use a self-signed certificate`
+|===
+
+NOTE: For AS2 endpoints, you must also configure at least one host keystore, as described in xref:manage-as2-host-keystores.adoc[]. 
+
+== Add a Host Certificate
+
+To add a host certificate:
+
+. In the sidebar, access the *<host>* page.
+. In the *Certificates* section, click *New*.
+. Select a certificate type.
+. Enter the settings for the selected certificate type:
++
+** <<public-certificate,Public Certificate>>
+** <<identity-key,Identity Key>>
++
+. Click *Save*.
+
+[[public-certificate]]
+=== Public Certificate Settings
+
+To configure settings for a public certificate:
+
+. In the *Certificate name* field, enter the name of the public certificate.
+. In the *Certificate* field, click *Choose file* and select a public certificate.
+
+[[identity-key]]
+=== Identity Key Certificate Settings
+
+To configure settings for an identity key certificate:
+
+. In the *Identity Key Name* field, enter the name of the identity key certificate.
+. In the *<host> Identity Key PEM File* field, click *Choose file* and select an identity key certificate.
+. In the *Key Passphrase* field, enter a passphrase for protecting the identity key certificate.
+. In  the *Expiration Date* field, optionally enter an expiration date for the certificate. This date is a reminder for you, but Anypoint Partner Manager doesn't enforce it.
+
+== Update a Host Certificate
+
+You can update a host certificate by selecting a different certificate file to associate with the certificate:
+
+. In the sidebar, access the *<host>* page.
+. In the *Certificates* section, click the name of the certificate that you want to update.
+. Click *Edit*.
+. Click *Choose File* and select a certificate file.
+. Click *Update*.
+
+== Remove a Host Certificate
+
+If a host certificate isn't associated with an endpoint, you can remove it from Anypoint Partner Manager:
```

Comment:
To remove a certificate from Anypoint Partner Manager:
.
.
.

You can remove a host certificate only if it isn't associated with an endpoint.

Reply:
I don't agree - while your wording is more consistent, it is important to have the caveat about host certificates associated with endpoints before the steps.